app/models/legal_aid_application.rb

```diff
@@ -7,9 +7,9 @@ class LegalAidApplication < ApplicationRecord
   has_one :benefit_check_result
 
   before_create :create_app_ref
+  before_save :set_consent_choice_date
 
-  attr_reader :proceeding_type_codes
-
+  attr_reader :proceeding_type_codes, :open_banking_consent, :consent_choice_timestamp
```

Comment:
You've added `open_banking_consent` and `consent_choice_timestamp` as attributes (in the migration). So they should not have a getter defined here.